mrmustard/lab/circuit_components_utils/b_to_ch.py

```diff
@@ -99,3 +102,6 @@ def inverse(self):
         ret._ansatz = super().inverse().ansatz
         ret._wires = ret.wires.dual
         return ret
+
+    def fock_array(self, shape: int | Sequence[int] | None = None) -> ComplexTensor:
+        raise NotImplementedError("BtoChar does not have a Fock representation.")
```

Comment:
Could something like this save you manually writing this 3x?
```suggestion
    def fock_array(self, shape: int | Sequence[int] | None = None) -> ComplexTensor:
        raise NotImplementedError(f"{self.__class__.__name__} does not have a Fock representation.")
```